docs/openapi/api-external-interop-v1.yaml

```
@@ -1,327 +1,394 @@
-          schema:
-            type: string
-            format: uuid
+      - name: eserviceId
+        in: path
+        required: true
+        description: The e-service name
+        schema:
+          type: string
+          format: uuid
+      - name: versionId
+        in: path
+        required: true
+        description: the e-service version
+        schema:
+          type: string
+          format: uuid
       requestBody:
         required: true
         content:
           application/json:
             schema:
-                $ref: '#/components/schemas/ChangeProbingFrequencyRequest'
+              $ref: '#/components/schemas/ChangeProbingFrequencyRequest'
       responses:
         '204':
-          description: >-
-            The update has been successfully completed
+          description: The update has been successfully completed
         '400':
-          description: >-
-            The provided input data is invalid
+          description: The provided input data is invalid
           content:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
         '404':
-          description: >-
-            The e-service hasn't been found
+          description: The e-service hasn't been found
           content:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
-        '500':                  
-          description: >-
-            A managed error has occured during the request elaboration
-          content:                        
+        '500':
+          description: A managed error has occured during the request elaboration
+          content:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
               example:
                 type: string
-                status: 500 
-                title: 'Internal Server Error'
-                detail: 'There is a problem processing the request on the server.'
-                traceId: 'string'
-                errors: [{'code':'000-1234', 'detail':'The server is currently down for maintenance. Please try again later.'}]
-                
-   '/eservices/searchEservices':
+                status: 500
+                title: Internal Server Error
+                detail: There is a problem processing the request on the server.
+                traceId: string
+                errors:
+                - code: 000-1234
+                  detail: The server is currently down for maintenance. Please try again later.
+  '/eserviceOperations/searchEservices':
     get:
       summary: Retrive e-services by filters
-      description: >-
-        Retrive e-services by filters : eserviceName , producerName, versionNumber , eservice-state 
+      description: 'Retrive e-services by filters : eserviceName , producerName, versionNumber
+        , eservice-state '
```

These details are already in the parameters section (and we will likely forget to update this description when parameters will be updated)
```suggestion
      description: 'Retrive e-services by filters'
```